pkg/virt-controller/watch/application.go

```diff
@@ -81,6 +83,7 @@ const (
 	defaultHost = "0.0.0.0"
 
 	launcherImage       = "virt-launcher"
+	exporterImage       = "virt-exporter"
```

Comment:
This is the default name that is set if the operator doesn't pass the exporter image here. We need to know the export server image name so we can create the right pod. We don't need to know the proxy name since the operator is creating that one.